projects/kit/src/lib/masks/time/time-mask.ts

```diff
@@ -24,7 +22,7 @@ export function maskitoTimeOptionsGenerator({
             TIME_FIXED_CHARACTERS.includes(char) ? char : /\d/,
         ),
         preprocessor: maskitoPipe(
-            createZeroPlaceholdersPreprocessor(mode),
+            createZeroPlaceholdersPreprocessor(),
```

Comment:
it requires no argument.

I think we can drop redundant JS closure and rename it to:
```suggestion
            zeroPlaceholdersPreprocessor,
```

Reply:
What do you think ?

Or is it better keeps it as it is (to be similar with other processors)?

Reply:
I like current varian, let's keep it as it is